charts/kuberpult/templates/frontend-service.yaml

```diff
@@ -92,6 +92,8 @@ spec:
           value: {{ .Values.git.author.email | quote }}
         - name: KUBERPULT_CDSERVER
           value: kuberpult-cd-service:8443
+        - name: KUBERPULT_VERSIONSERVER
+          value: kuberpult-manifest-repo-export-service:8443
```

Comment:
this is only correct if the db is enabled. If not, we need to use the same as above (kuberpult-cd-service:8443)